sdk/ai/azure-ai-inference/tsp-location.yaml

```diff
@@ -1,5 +1,5 @@
-commit: 70d146a18939fcfd36661adc0bb123046cd3d19b
+commit: f987e92b69220bc116babb3a976a8777106486bd
 additionalDirectories: []
-repo: Azure/azure-rest-api-specs
+repo: trangevi/azure-rest-api-specs
```

Comment:
trangevi -> Azure?

Reply:
Currently it's pointing to my fork while I'm working on the typespec merge. That PR will go in first, at which point I'll switch this over to pointing to the Azure repo and update the commit pointer, and then merge this